src/Converter.java

```diff
@@ -9,13 +11,64 @@ public static String convertString(String markdownStr) {
     }
 
     private static String convertGeneralFormatting(String str) {
-        String altered = null;
-        return altered;
+      String altered = null;
+      return altered;
     }
 
     private static String convertItalics(String str) {
```

Comment:
A lot of methods we need will replace "surrounding" markdown features (e.g. bolding and italics) by surrounding the same text with HTML tags.
We should turn this into a general function that takes in the MD feature to look for and the HTML tag to replace it with.

Reply:
All set I hope! As for two spaces...